Tests/OpenAPIRuntimeTests/Test_Runtime.swift

```diff
@@ -98,6 +102,10 @@ class Test_Runtime: XCTestCase {
         """#
     }
 
+    var testStructURLFormString: String {
+        "age=3&name=Fluffz&type=Dog"
```

Comment:
Please make one of the strings have a space, and a special character, to verify escaping.